pointgrey_camera_driver/CMakeLists.txt

```diff
@@ -29,11 +29,11 @@ catkin_package(CATKIN_DEPENDS
 # contains them. We work around this by downloading the archive directly from
 # their website during this step in the build process.
 find_library(Spinnaker_LIBRARIES Spinnaker)
-if(NOT Spinnaker_LIBRARIES)
+# if(NOT Spinnaker_LIBRARIES)
```

Comment:
Not sure if this is intentional or part of some testing, either way I guess you'll revisit this later, right?

Reply:
I commented this out so that the RZR's with the old Spinnaker driver would get updated with the latest version that I've added to the cmake file.